sympy/core/symbol.py

```diff
@@ -226,9 +226,20 @@ def _diff_wrt(self):
         """
         return True
 
+    @staticmethod
+    def _validate_assumptions(assumptions):
```

Comment:
It would make more sense to have this in `_sanitize`, since every call is followed by `_sanitize` anyway.